### DISCUSSION TITLE
Next JS 13.5 - Ant Design FOUC problem

### DISCUSSION THREAD

**Discussion by alibugatekinn** [ant-design/ant-design#44984] [Q&A] [answered]
i am using ant design 5.9.1 version. but there is a big problem is Flash of Unstyled Content (famous problem FOUC). How can I fix that? i am asking other than opacity and visibility fixes.

**Comment by xiaozheqiu** [comment]
I also encountered this issue. Although it can be extracted through extractStyle, the content in the header of CSS is too large

**Comment by arvinxx** [comment]
https://ant-design.github.io/antd-style/guide/ssr#%E5%9C%A8-nextjs-%E4%B8%AD%E9%9B%86%E6%88%90

**Comment by arvinxx** [comment, marked as answer]
Here is an project that use antd in really world with nextjs 13.5: https://chat-preview.lobehub.com/

it's github : https://github.com/lobehub/lobe-chat

you can learn how to fix FOUC problem with this project.
